backend/geonature/core/gn_synthese/routes.py

```diff
@@ -313,6 +440,68 @@ def get_one_synthese(permissions, id_synthese):
     if not synthese.has_instance_permission(permissions=permissions):
         raise Forbidden()
 
+    sensitivity, _ = split_blurring_permissions(permissions=permissions)
+
+    sensitivity = list(sensitivity)
+
+    # If sensitivity permissions and obs sensitive. FIXME: HardCoded "0"?
+    if len(sensitivity) != 0 and synthese.nomenclature_sensitivity.cd_nomenclature != "0":
```

Comment:
J’ai l’impression que tu floutes même si tu as également accès à l’observation via une permission précise.
Je rajouterai à la condition du if ``and not synthese.has_instance_permissions(without_blurring_permissions)`` (le deuxième retour de ``split_blurring_permissions``).

Reply:
Et du coup le ``len(sensitivity) != 0`` devient inutile : si tu n’as pas accès à l’obs avec les permissions précises, mais que tu y as avec toutes les permissions, c’est forcément que tu y as accès via une permission floutage.

Reply:
Merci beaucoup pour ton retour @bouttier !
Ah oui effectivement bien vu !
J'ignore les permissions non sensibles alors qu'elles peuvent exister et donc dire que j'ai accès à cette observation de manière précise ! Je corrige ça ! Merci encore !

Reply:
C'est corrigé par https://github.com/PnX-SI/GeoNature/pull/2571/commits/2580b981caef2b8d2a3165ae81e4c4c958539d45 !
https://github.com/PnX-SI/GeoNature/pull/2571/files#diff-5fb8d19676b01cc7c0b77799d74e25c6a6ff510372180a633a2ecd1682c3c4d9R448-R451